docs/modules/servers/pages/distributed/configure/listeners.adoc

```diff
@@ -17,6 +17,8 @@ to get some examples and hints.
 
 == Configuration
 
+The <executeGroupListeners> controls whether to launch group mailbox listener consumption. Defaults to true. Use with caution.
```

Comment:
In this case "Use with caution" is not very clear, even to me. You should be more explicit telling that it should be set to false only if you forecast to have an other James server in the cluster set to true.